docs/content/users/usage/commands.md

```diff
@@ -680,19 +680,19 @@ Example:
 ddev import-db
 
 # Import the `.tarballs/db.sql` dump to the project database
-ddev import-db --src=.tarballs/db.sql
+ddev import-db --file=.tarballs/db.sql
 
 # Import the compressed `.tarballs/db.sql.gz` dump to the project database
-ddev import-db --src=.tarballs/db.sql.gz
+ddev import-db --file=.tarballs/db.sql.gz
 
-# Import the compressed `.tarballs/db.sql.gz` dump to a `newdb` database
-ddev import-db --target-db=newdb --src=.tarballs/db.sql.gz
+# Import the compressed `.tarballs/db.sql.gz` dump to a `other_db` database
+ddev import-db --database=other_db --file=.tarballs/db.sql.gz
 
 # Import the `db.sql` dump to the project database
-ddev import-db <db.sql
+ddev import-db < db.sql
 
-# Import the `db.sql` dump to a `newdb` database
-ddev import-db newdb <db.sql
+# Import the `db.sql` dump to a `some-project` project
+ddev import-db some-project < db.sql
```

Comment:
If I’m following correctly, this is importing to the `db` database in a custom project. So I’d prefix with `my-`, a convention we’re generally using to indicate user-generated handles + details, and adjust the description to clarify:

```suggestion
# Import the `db.sql` dump to the `my-project` default database
ddev import-db my-project < db.sql
```

Reply:
Done